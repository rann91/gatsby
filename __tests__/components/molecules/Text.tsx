import { render } from '@testing-library/react'
import React from 'react'
import { Text } from 'theme-ui'
import component from '../../fixtures/component-text.json'

describe('Text', () => {
  it('renders correctly', () => {
    const { container } = render(<Text {...component} />)

    expect(container).toMatchSnapshot()
  })
})
