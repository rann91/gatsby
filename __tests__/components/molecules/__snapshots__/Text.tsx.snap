// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
<div>
  <div
    class="css-vurnku"
    id="5f2aa6d014359808497bba19"
  />
</div>
`;
